mappings/net/minecraft/util/JsonHelper.mapping

```diff
@@ -210,3 +210,5 @@ CLASS net/minecraft/class_3518 net/minecraft/util/JsonHelper
 	METHOD method_34933 getShort (Lcom/google/gson/JsonObject;Ljava/lang/String;)S
 		ARG 0 object
 		ARG 1 element
+	METHOD method_37165 deserializeJsonArray (Ljava/io/Reader;)Lcom/google/gson/JsonArray;
```

Comment:
Maybe just `deserializeArray` since it's in `JsonHelper` class.
```suggestion
	METHOD method_37165 deserializeArray (Ljava/io/Reader;)Lcom/google/gson/JsonArray;
```